### PR TITLE
Add EIP: Increase MIN_BASE_FEE_PER_BLOB_GAS

### PR DESCRIPTION
**Pull request by MaxResnick**
**ATTENTION: ERC-RELATED PULL REQUESTS NOW OCCUR IN [ETHEREUM/ERCS](https://github.com/ethereum/ercs)**

--

When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-draft_increase_blob_base_fee.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+eip: <to be assigned>
```
Comment (abcoathup) [inline EIPS/eip-draft_increase_blob_base_fee.md:2]:
```suggestion
eip: 7762
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

File: EIPS/eip-draft_increase_blob_base_fee.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+title: Increase Blob Base Fee
+description: Adjust the MIN_BASE_FEE_PER_BLOB_GAS to speed up price discovery on blob space
+author: <Max Resnick (@MaxResnick)>
+discussions-to: <URL>
```
Comment (abcoathup) [inline EIPS/eip-draft_increase_blob_base_fee.md:6]: Please create a discussions topic in Eth Magicians using the template in the EIPs category:
https://ethereum-magicians.org/c/eips/5

**Review by Magken91 — APPROVED** [review]

**Review by Allzz77123 — APPROVED** [review]

**Review by Allzz77123 — APPROVED** [review]

File: EIPS/eip-7762.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+---
+eip: <7762>
```
Comment (abcoathup) [inline EIPS/eip-7762.md:2]:
```suggestion
eip: 7762
```

File: EIPS/eip-7762.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+title: Increase MIN_BASE_FEE_PER_BLOB_GAS
+description: Adjust the MIN_BASE_FEE_PER_BLOB_GAS to speed up price discovery on blob space
+author: <Max Resnick (@MaxResnick)>
+discussions-to: <https://ethereum-magicians.org/t/eip-7762-increase-min-base-fee-per-blob-gas/20949>
```
Comment (abcoathup) [inline EIPS/eip-7762.md:6]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-7762-increase-min-base-fee-per-blob-gas/20949
```

**Review by jamesavechives — APPROVED** [review]

File: EIPS/eip-7762.md (hunk 4 above)
Comment (lightclient) [reply EIPS/eip-7762.md]: same here

File: EIPS/eip-7762.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+## Specification
+
+The only specification change introduced by this EIP is setting MIN_BASE_FEE_PER_BLOB_GAS to 2**25:
+
+```diff
++ MIN_BASE_FEE_PER_BLOB_GAS  =  2**25
+- MIN_BASE_FEE_PER_BLOB_GAS = 1
+```
+
+## Rationale
```
Comment (lightclient) [inline EIPS/eip-7762.md:33]:
```suggestion
## Rationale

```

File: EIPS/eip-7762.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+63070646/(3 * 2**17) = 160.396947225
+```
+
+The closest power of 2 to the corresponding reserve price would be `MIN_BASE_FEE_PER_BLOB_GAS = 2**27`. Out of an abundance of caution, we will go with `MIN_BASE_FEE_PER_BLOB_GAS = 2**25` to ensure that even if the price of ETH rises significantly, the reserve price will not be set too high. This value corresponds to a minimum blob price of ~1 cent at today's prices (2,445.77$ ETH/USDC). Further, decreasing the `MIN_BASE_FEE_PER_BLOB_GAS` beyond `2**25` would slow down price discovery without a significant decrease in the price of blobs when the network is unsaturated. 
+
+Below you will find a plot provided by @dataalways showing the fraction of type 3 transaction fees that are paid in blob base fees for different values of `MIN_BASE_FEE_PER_BLOB_GAS`. Note that even after the proposed change, for historical values of l1 gas, the price of blobs would have been dominated by the price of the L1 gas.  
+
+![Base Fee 1](base_fee_1.png)
+
+<div style="page-break-after: always;"></div>
```
Comment (lightclient) [inline EIPS/eip-7762.md:50]: inline html is not allowed. please remove

File: EIPS/eip-7762.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
+---
+eip: 7762
+title: Increase MIN_BASE_FEE_PER_BLOB_GAS
+description: Adjust the MIN_BASE_FEE_PER_BLOB_GAS to speed up price discovery on blob space
+author: <Max Resnick (@MaxResnick)>
```
Comment (lightclient) [inline EIPS/eip-7762.md:5]:
```suggestion
author: Max Resnick (@MaxResnick)
```

File: EIPS/eip-7762.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+---
+
+
+## Backwards Compatibility
+
+This EIP is not backwards compatible and requires a coordinated upgrade across all clients at a specific block number.
+
+## Security Considerations
+
+- Rollups that use blobs as a data availability layer will need to update their posting strategies.
```
Comment (lightclient) [inline EIPS/eip-7762.md:65]:
```suggestion
Rollups that use blobs as a data availability layer will need to update their posting strategies. 
```

**Comment by lightclient** [comment]
If you would like this EIP to be merged, please undraft the PR.

**Review by XstRUhayabusa — CHANGES_REQUESTED** [review]
Dankeschön

File: EIPS/eip-7762.md (hunk 7 above)
Comment (JanKortes212) [reply EIPS/eip-7762.md:5]: ok

**Comment by lightclient** [comment]
@MaxResnick can you please fix the CI issues so we can merge this EIP.

**Comment by MaxResnick** [comment]
@lightclient should be good to go

File: EIPS/eip-7762.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+This EIP is not backwards compatible and requires a coordinated upgrade across all clients at a specific block number.
+
+## Security Considerations
+
+Rollups that use blobs as a data availability layer will need to update their posting strategies. 
+
+## Copyright
+
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```
Comment (macfarla) [inline EIPS/eip-7762.md:68]:
```suggestion
Copyright and related rights waived via [CC0](../LICENSE.md).
```

File: base_fee_225.png
Comment (macfarla) [inline base_fee_225.png:1]: the images are added twice in separate locations, perhaps unintentional?

**Comment by g11tech** [comment]
CI still not passing

**Comment by MaxResnick** [comment]
> CI still not passing

Passing now

**Review by g11tech — APPROVED** [review]

**Pull request merged**